konnector.js

```diff
@@ -72,6 +72,7 @@ function login(requiredFields, billInfos, data, next) {
     logger.info('Signing in')
     request(signInOptions, (err, res) => {
         if (err) {
+            logger.error('LOGIN_FAILED')
```

Comment:
There already is a log saying that the signin failed ?

Reply:
Yes, but to have it work as intented, we need to log explicitly the message `LOGIN_FAILED` as error and before any other error message.

Reply:
Could you explain please ? I should be able to log whatever I want.

Reply:
Of course. To be able to detect a login error and handle this error the best way as possible, the only way for the stack to know that the login failed is to analyse konnector's output, the logs. When the konnector fails and stops, the stack parse the output and retrieve the first occurence of a log with the level `error`. So to have the behavior we need, we have to first log a `LOGIN_FAILED` message.

This will be improved in the future and konnector developers will be able to log whatever they want whenever they want. :)

Reply:
I expect konnectors to handle the state through what I pass to `next()`, not from the logs…

Reply:
I totally agree. The thing is for now it's the only way for the stack to retrieve this information, and it is very useful to cozy-collect for providing the best UX to user.

Reply:
(I should have been more explicit in the description of my PR, I updated materielnet along with a bunch of another connectors)